### ISSUE TITLE
Client IP is always 'null'

### ISSUE THREAD

**Issue by Joschi84** [elysiajs/elysia#599]
### What version of Elysia.JS is running?

Elysia: command not found

### What platform is your computer?

Linux 5.15.146.1-microsoft-standard-WSL2 x86_64 x86_64

### What steps can reproduce the bug?

Hello Community,

I'm trying to get the client IP Address, but i only always get 'null'. What I am doing wrong?
Hopefully someone can help.
Elysia is working well (even if 'Elysia --revision' gives      Elysia: command not found).
Bun is on the newest Version (1.1.3). And Elysia should be also very up-to-date.
Here is the code:

```
import { Elysia } from "elysia";

const app = new Elysia()
  .use(app => app.derive(({ request }) => ({ ip: app.server?.requestIP(request) })))
  .post("/test", async ({ body, ip }) => {
      console.log(body, ip) 
  })
  .listen(2000)
```

### What is the expected behavior?

Getting client ip address.

### What do you see instead?

null

**Comment by kidqueb** [comment]
Your snippet works for me when logging `ip`, but it is currently logging `id`.

**Comment by Joschi84** [comment]
My fault. I edited it. But this was just a typing error here in the browser. For me, the problem remains unclear.  Any further ideas?

**Comment by SaltyAom** [comment]
Blocking on https://github.com/oven-sh/bun/issues/6613

**Comment by kravetsone** [comment]
[Fixed by Bun@1.1.27](https://bun.sh/blog/bun-v1.1.27#fixed-server-requestip-after-await)

```ts
import { Elysia } from "elysia";

const app = new Elysia();
const port = 3000;

app.post("/", (context) => {
    return `Your IP address: ${
        app.server?.requestIP(context.request)?.address || null
    }`;
});

app.listen(port, () => {
    console.log(`Running on port ${port}`);
});

```
![image](https://github.com/user-attachments/assets/5cdfde26-e9c4-441f-acf0-81a12fe0c57c)
